src/ads/RewardedAd.ts

```diff
@@ -158,4 +150,37 @@ export class RewardedAd extends MobileAd implements MobileAdInterface {
     }
     return this._googleMobileAds.native.rewardedShow(this._requestId, this._adUnitId, options);
   }
+
+  addAdEventsListener<T extends AdEventType | RewardedAdEventType>(
```

Comment:
In the hypothetical change where this is implemented in the superclass, then this still gets an implementation here but it is just to define the new types, then it calls the super class method for validation and actual addition of listener etc